sdk/iot/modelsrepository/CODE_STRUCTURE.md

```diff
@@ -0,0 +1,71 @@
+## index.ts
```

Comment:
what is this file? Are you committing to documenting every source file? Wouldn't it make more sense to document them in comments in the files themselves?